app/ui/lib/SettingsGroup.tsx

```diff
@@ -21,9 +21,20 @@ type Props = {
   /** String action is a link */
   cta: string | (() => void)
   ctaText: string
-}
+} & (
+  | { secondaryCta: string | (() => void); secondaryCtaText: string }
+  | { secondaryCta?: undefined; secondaryCtaText?: undefined }
+)
 
-export const SettingsGroup = ({ title, docs, children, cta, ctaText }: Props) => {
+export const SettingsGroup = ({
+  title,
+  docs,
+  children,
+  cta,
+  ctaText,
+  secondaryCta,
+  secondaryCtaText,
+}: Props) => {
```

Comment:
I'm thinking this is an excessive level of abstraction for something that's used in one place. I think I can break this up into a few reusable containers and cut this down a lot.